src/components/leaderboards/Leaderboard/index.js

```diff
@@ -20,11 +21,12 @@ module.exports = React.createClass({
     charitySlug: React.PropTypes.string,
     charityUid: React.PropTypes.string,
     country: React.PropTypes.oneOf(['au', 'ie', 'nz', 'uk', 'us']),
-    type: React.PropTypes.oneOf(['team', 'individual']),
+    type: React.PropTypes.oneOf(['individual']),
```

Comment:
yeah but for this component, it can ONLY be 'individual'. Anything other than that will throw an error. If it *has* to be 'individual', remove this as a prop and just bake it into the endpoint url.